### DISCUSSION TITLE
HTTP2 Supports?

### DISCUSSION THREAD

**Discussion by dalisoft** [uNetworking/uWebSockets.js#178] [Q&A] [answered]
Hi @alexhultman 

Is uWebSockets.js supports HTTP2?
If yes, there any flag need or supports out-of-the-box?

Thanks.

Related issue: [#27](https://github.com/dalisoft/nanoexpress/issues/27)

**Comment by ghost** [comment]
No that is not a goal for this project right now

**Comment by dalisoft** [comment]
Thanks for reply. I just wanna to know and now know. Without http2 is this project is very fast and flexible

**Comment by ghost** [comment, marked as answer]
Http is really just a secondary thing here. Http2 brings no improvements to WebSockets, in fact it only adds more overhad and less portability. The QuicTransport is a much better modern feature to look forward to.

**Reply by dalisoft** [reply]
Yes, you absolutely right.
Also QUIC support would be nice.
